docn/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 project(docn Fortran)
 set(SRCFILES ocn_comp_nuopc.F90
-   	     docn_datamode_copyall_mod.F90
+	     docn_datamode_copyall_mod.F90
```

Comment:
align these?

Reply:
Everything is now re-aligned under SRCFILES.